docs/usage/faq.md

```diff
@@ -170,29 +176,63 @@ As above, but apply a `groupName`, e.g.
 
 Set the `branchName`, `commitMessage`, `prTitle` or `prBody` configuration options:
 
-```
+```json
 "branchName": "vroom/{{depName}}-{{newMajor}}.x",
 "commitMessage": "Vroom vroom dependency {{depName}} to version {{newValue}}",
 "prTitle": "Vroom {{depName}},
 ```
 
+<!-- The example looks vague to me as well, should we use a proper JSON example? -->
```

Comment:
This is a bit vague to me, could we maybe put the `branchName`, `commitMessage` and `prTitle` properties in a proper `renovate.json` config look-a-like or something, so that you know where those properties should go?

Reply:
Yes, let's remove this example completely